Purchases/Misc/RCCrossPlatformSupport.h

```diff
@@ -58,3 +61,11 @@
 #else
 #define PURCHASES_INITIATED_FROM_APP_STORE_AVAILABLE 0
 #endif
+
+#if TARGET_OS_IOS || TARGET_OS_TV
+#define IS_APPLICATION_BACKGROUNDED UIApplication.sharedApplication.applicationState == UIApplicationStateBackground
+#elif TARGET_OS_OSX
+#define IS_APPLICATION_BACKGROUNDED !NSApplication.sharedApplication.isActive
```

Comment:
in my testing, `isActive` is returning `nil`. I'm going to be conservative here and just return `YES` for mac, since mac apps have a different lifecycle anyway.